exploration/string-selection-formatting.md

```diff
@@ -0,0 +1,114 @@
+# Selection and Formatting on String Values
+
+Status: **Proposed**
+
+<details>
+	<summary>Metadata</summary>
+	<dl>
+		<dt>Contributors</dt>
+		<dd>@aphillips</dd>
+		<dt>First proposed</dt>
+		<dd>2024-02-16</dd>
+		<dt>Pull Request</dt>
+		<dd><a href="https://github.com/unicode-org/message-format-wg/pull/660">#660</a></dd>
+	</dl>
+</details>
+
+## Objective
+
+Define how selection and formatting of string literals takes place.
+
+## Background
+
+This design doc covers the text needed in the registry for selection and formatting of
+string values.
+This includes functionality to match or replace that found in MF1's `SelectFormat`.
+
+## Use-Cases
+
+As a user, I wish to select the pattern to use based on an enumerated list of values.
+
+As a user, I wish to insert strings into a pattern.
+
+As a user, I wish to insert data values that do not have a custom or built-in formatting
+function into a pattern, relying on the data value or object's ability to stringify itself.
+
+
+## Requirements
+
+- Enable `SelectFormat`
+
+## Constraints
+
+ICU MF1 messages using `select` should be representable in MF2.
+
+## Proposed Design
+
+## String Selection and Formatting
+
+### Functions
+
+The following functions are provided:
+
+The function `:string` provides string selection and formatting.
+
+### Operands
+
+The _operand_ is any literal or an implementation-defined set of string or
+character-sequence types.
+
+> [!NOTE]
+> This should probably include individual character types, such as `char`.
+
+In addition, implementations MAY perform formatting and selection on 
+`operand` values that do not otherwise have a formatting function registered, 
+although emitting an `Unsupported Expression` error is RECOMMENDED.
+
+### Options
+
+The function `:string` has no options.
+
+> [!NOTE]
+> Proposals for string transformation options or implementation
+> experience with user requirements is desired in the Technical Preview.
```

Comment:
We should be consistent in our naming, and also match how CLDR refers to this phase as a "tech preview".

```suggestion
> experience with user requirements is desired during the Tech Preview.
```